node-operators/block-producers/onboarding/run-a-block-producer/block-producer-docker.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Run a Block Producer Using Docker
+description: Learn how to set up and run a block producer (aka collator or sequencer) for Tanssi Appchains using Docker to participate in the protocol and earn rewards.
+---
+
+# Run a Block Producer in Tanssi Using Docker
+
+## Introduction {: #introduction }
+
+--8<-- 'text/node-operators/block-producers/run-a-block-producer/intro.md'
+
+In this guide, you'll learn how to spin up a Tanssi block producer to be part of the shared pool of sequencers using [Docker](https://www.docker.com/){target=\_blank} on a Linux computer. However, it can be adapted to other operating systems.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+--8<-- 'text/node-operators/installing-docker.md'
+
+### Pulling the Docker Image {: #pulling-docker-image }
+
+A Docker image is built and published in every release, containing all the necessary dependencies a Tanssi block producer requires and the binary file itself.
+
+A Docker image combines the binary corresponding to the latest stable release of the [client node](/learn/framework/architecture/#architecture){target=\_blank}, along with the Tanssi orchestrator specification file.
+
+The following command to pull the Docker image:
+
+```bash
+docker pull moondancelabs/parachain-dancebox
```

Comment:
I talked with Gorka and he said this is OpsLayer specific image.

The public facing image is:

```
docker pull moondancelabs/tanssi:v0.4
```

We should check here if there is a `skylake` and `zen3` file etc

Reply:
Please also check RPC docker image

Reply:
updated/checked the image's names and entrypoints, and unified the commands in both rpc and collators